README.md

```diff
@@ -113,159 +98,61 @@ The tool works with the following types of asset files (also referred to as _ass
 
 NOTE: Quicktime (`.mov`) format is not yet fully supported.
 
-## Usage
```

Comment:
Important caveat for my review: The diffs are really hard to follow here, so I am not checking to ensure that all of Peleus' original content is accounted for somewhere. I'm only reading / proofreading the new content.

Reply:
Yes, I agree the diffs are hard to follow.  Perhaps the best option is to view the files directly in the branch: https://github.com/contentauth/c2pa-attacks/blob/doc-edits/README.md.  This won't directly tell you if everything is there vs. the original, but you can more easily read it to see if it's complete and compare it side-by-side with the original.   Certainly, my intent is to keep all the original information, but make it more clear and concise.